styles.css

```diff
@@ -0,0 +1,95 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+body {
+  height: 812px;
+  width: 375px;
+  background-color: #fff;
+}
+
+.nav {
+  display: block;
+  height: 70px;
+  margin: 15px;
+  padding: 0;
+  top: 0;
+  width: 96%;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 10px 15px;
+}
+
+.logo {
+  height: 45px;
+  width: 7%;
+  float: left;
+  color: #36b37e;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.menubar .fa-bars {
+  float: right;
+  font-size: 30px;
+  margin-right: 1.5%;
+  color: #000;
+}
+
+.flexcontainer {
+  height: 80%;
+  display: flex;
+  margin: 65px 2px;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+}
+
+.flexitem {
+  margin: auto;
+  height: 20%;
+  width: 90%;
+  justify-content: center;
+}
+
+.heading {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 35px;
+  text-align: center;
+}
+
+.heading h1 {
+  color: #172b4d;
+}
+
+.heading b {
+  font-weight: 600;
+  color: #36b37e;
+}
+
+.paragraph {
+  height: 10%;
+  width: 80%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.socialmediaicons {
+  height: 2%;
+  width: 40%;
+  align-self: center;
+  font-size: 20px;
+  margin-left: 35%;
+}
+
+.fa {
+  color: #505f79;
+  margin: 0% 0.5%;
```

Comment:
Please i'd suggest you adjust this icons and add some space between them as shown in the design 😄